recipes/cuda-nvcc/nvcc.profile.patch

```diff
@@ -0,0 +1,14 @@
+--- bin/nvcc.profile        2021-07-06 11:22:41.714913066 -0700
++++ bin/nvcc.profile.new    2021-07-06 11:22:33.467108854 -0700
+@@ -6,9 +6,9 @@
+ LD_LIBRARY_PATH += $(TOP)/lib:
+ PATH            += $(TOP)/$(_NVVM_BRANCH_)/bin:$(_HERE_):
+
+-INCLUDES        +=  "-I$(TOP)/$(_TARGET_DIR_)/include" $(_SPACE_)
++INCLUDES        +=  "-I$(TOP)/include" $(_SPACE_)
+
+-LIBRARIES        =+ $(_SPACE_) "-L$(TOP)/$(_TARGET_DIR_)/lib$(_TARGET_SIZE_)/stubs" "-L$(TOP)/$(_TARGET_DIR_)/lib$(_TARGET_SIZE_)"
++LIBRARIES        =+ $(_SPACE_) "-L$(TOP)/lib/stubs" "-L$(TOP)/lib"
```

Comment:
Why do this? This would make it hard to enable cross compiling

Reply:
Our packages have been installing directly into `lib` and `include` directories, likely to follow what `cudatoolkit` did. It doesn't appear there was a plan to add cross packages at this point.

Reply:
Yeah it would be good to add cross-compiling if we can. This has recently come up a few times as a user request. Here's a few references:

* https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/209
* https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/210

Reply:
Is there any recommendation on where to move the libraries to? Other installer formats just use something like `targets/<multiarch>/lib` and create a `lib` symlink to the host target, which only works when cuda doesn't share the `lib` directory.

Reply:
Maybe it would make sense to put these in `${CONDA_BUILD_SYSROOT}/lib`. Though thoughts from others welcome here

Somewhat related question, do we include some sort of `libcuda.so` stub library as well? We've made [use of the stub library before during builds]( https://github.com/conda-forge/nvcc-feedstock/blob/9c8e43393d9c798c52a9d8e56785920ef767cabb/recipe/linux/activate.sh#L97-L110 ), which has been pretty helpful.

Reply:
We can add them to `lib/targets/<multiarch>/lib` and do symlinks for each file in all the packages.

Reply:
Removed this patch now. Files should now be under `targets/<multiarch>/lib` and `targets/<multiarch>/include` with symlinks to each file